### DISCUSSION TITLE
How to see what your html file code looks like on a web page

### DISCUSSION THREAD

**Discussion by Adam-Brian** [community/community#106893] [New to GitHub]
I am brand new to github.  I have just downloaded the Visual Studio Code to edit my code on my desktop pc but I thought I would give codespaces a try as it would enable me to use Visual Studio Code from a web browser on any device on the go.  When i am writing some code for example some html, if I want to view how it would look like on a web page, how do I do this?  When I use the Visual Studio Code app on my desktop pc, I just right click on the html file in the editor, select copy path and then paste the link into web browser.  There doesn't seem to be any option to do this when using codespaces.

**Comment by SaraRasoulian** [comment]
Install `Live Server` extension on your VSCode, then click F5 to run your code

**Reply by Adam-Brian** [reply]
Thanks for the quick reply.

**Comment by ZephyrusBlaze** [comment]
To view how your HTML code looks like on a web page using Visual Studio Code or Codespaces, you can install the Live Server extension. Once installed, you can either click "Go Live" from the status bar, right-click on an HTML file and select "Open with Live Server," or use keyboard shortcuts like (alt+L, alt+O) to start the server. This will open your HTML file in a web browser, allowing you to see the live preview. [Live Server Extension](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer).

**Reply by Adam-Brian** [reply]
Thanks for the quick reply.

**Reply by Adam-Brian** [reply]
Where did you get the link from for Live Server?  I went to visualstudio.microsoft.com but can't find the marketplace anywhere.

**Reply by ZephyrusBlaze** [reply]
You can find the Live Server extension by searching for it in the Extensions view within Visual Studio Code. Just open the Extensions view by clicking on the square icon on the sidebar or by pressing Ctrl+Shift+X, then type "Live Server" into the search bar and install it from there.

**Reply by Adam-Brian** [reply]
ok thanks.  Is there a link to marketplace.visualstudio.com at the site https://visualstudio.microsoft.com/?

**Reply by ZephyrusBlaze** [reply]
![Screenshot_20240209_175801_Brave](https://github.com/community/community/assets/113236396/cee0dcec-8ffb-44c4-b4fe-3d8e6a4652e9)
This? https://marketplace.visualstudio.com/

**Reply by Adam-Brian** [reply]
that's great, thanks

**Comment by Adam-Brian** [comment]
After I installed it, in codespaces I can't see the Go Live anywhere in codespaces, only see it in the Visual Studio Code which i downloaded to desktop.  How do i use it in codespaces?
